boards/framework/laptop16_keyboard/framework_laptop16_keyboard.dts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2025 Framework Comptuer Inc
+ * SPDX-License-Identifier: Apacha-2.0
+ */
+
+/dts-v1/;
+
+#include <raspberrypi/rpi_pico/rp2040.dtsi>
+#include <zephyr/dt-bindings/pwm/pwm.h>
+#include "framework_laptop16_keyboard-pinctrl.dtsi"
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,flash-controller = &ssi;
+		zephyr,code-partition = &code_partition;
+	};
+
+	aliases {
+		led0 = &caps_led;
+		pwm-led0 = &white_backlight_pwm;
+	};
+
+	zephyr,user {
+		io-channels = <&adc 2>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		caps_led: caps_led {
```

Comment:
not formatted properly, see https://docs.zephyrproject.org/latest/contribute/style/devicetree.html and fix all issues in whole PR